hibernate-community-dialects/src/test/java/org/hibernate/community/dialect/RowsLimitHandlerTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.community.dialect;
```

Comment:
Shouldn't this be in a test package?

Reply:
There are other tests already exist in this package, I think it's better to keep it.

Reply:
Oh, well. That doesn't seem quite right to me.

@beikov is there a good reason the tests for the community dialects are in the same package as the dialect classes? Shouldn't they be somewhere like `org.hibernate.community.dialect.test` or whatever?

Reply:
Having tests in the same package as the Dialect under test allows to test package-private methods, which I suppose was the original reason for doing this. When we moved the classes to the community dialects module, we didn't change the package for tests apparently. Either is fine IMO, though if you want to change it, I think I'd prefer the package you are proposing.